src/components/Card/Card.jsx

```diff
@@ -0,0 +1,65 @@
+import imgg from "./dummy-pot-plant.png";
+
+export default function Card(props) {
+  const priAction =
+    props.priAction ??
+    (() => {
+      console.log("No Primary Action Defined");
+    });
+  const secAction =
+    props.secAction ??
+    (() => {
+      console.log("No Seconday Action Defined");
+    });
+  const itemDetails = props.itemDetails ?? {name: "Amazing Pot"};
+  const itemName = itemDetails.name;
+  console.log("Inside card: ");
+  console.log(props);
```

Comment:
Remove all the console.logs.

Reply:
will be removed in upcoming refactoring.